statusmonitor/main.go

```diff
@@ -415,48 +415,53 @@ func getControlStatusFromResponse(statusBody []byte) *contrailOperatorTypes.Cont
 		}
 	}
 
-	numDownStaticRoutes := "0"
-	numStaticRoutes := "0"
-	numUpBgpPeer := "0"
-	numBgpPeer := "0"
 	numUpXMPPPeer := "0"
 	numRoutingInstance := "0"
 	staticRoutes := contrailOperatorTypes.StaticRoutes{}
 	bgpPeer := contrailOperatorTypes.BGPPeer{}
 	state := "down"
 
 	if controlUVEStatus != nil {
-		if len(controlUVEStatus.BgpRouterState.NumDownStaticRoutes) > 0 && len(controlUVEStatus.BgpRouterState.NumDownStaticRoutes[0]) > 0 && controlUVEStatus.BgpRouterState.NumDownStaticRoutes[0][0].Text != "" {
-			numDownStaticRoutes = controlUVEStatus.BgpRouterState.NumDownStaticRoutes[0][0].Text
+		numDownStaticRoutes, err := controlUVEStatus.BgpRouterState.NumDownStaticRoutes.Status()
+		if err != nil {
+			log.Fatalf("numDownStaticRoutes", err)
```

Comment:
You changed the code logic by adding Fatal here. Please change it to Info/Debug

Reply:
Removed errors as there were none